app-office/worklog-assistant/worklog-assistant-2.1.3577.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit unpacker
+
+DESCRIPTION="Hassle-free time tracking for JIRA self-hosted and OnDemand"
+HOMEPAGE="https://worklogassistant.com"
+
+SRC_URI="https://worklogassistant.com/downloads/${PN}-v2_${PV}_amd64.deb"
+
+LICENSE="worklog-assistant"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}"
+RESTRICT="strip"
+
+src_prepare() {
+	rm etc/ -r || die
+	rm _gpgbuilder || die
+
+	eapply_user
```

Comment:
instead of calling `eapply_user`, call `default`, and call it at the beginning of the function